### PR TITLE
Use Ipopt_jll binaries on Julia pre v1.3 too

### PR DESCRIPTION
**Pull request by blegat**
@dpo What is the use of `Libdl.dlpath` in:
https://github.com/JuliaOpt/Ipopt.jl/blob/5efbc4e15ac2b565b6c7880ed404159a6289a890/src/Ipopt.jl#L11
`libipopt` is already an absolute path.

### CODE REVIEW

**Comment by dpo** [comment]
Not sure it's necessary. I was mostly trying to preserve the variable names in Ipopt.jl.

**Comment by dpo** [comment]
ps: there's an updated https://github.com/JuliaBinaryWrappers/Ipopt_jll.jl/releases

**Comment by blegat** [comment]
> ps: there's an updated https://github.com/JuliaBinaryWrappers/Ipopt_jll.jl/releases

Indeed, that's the one I used to generate the `build.jl`.

File: deps/build.jl [hunk 1]
```diff
@@ -1,4 +1,8 @@
-using BinaryProvider # requires BinaryProvider 0.3.0 or later
+if VERSION >= v"1.3" && !haskey(ENV, "JULIA_CLP_LIBRARY_PATH")
```
Comment (amontoison) [inline deps/build.jl:1]: `JULIA_IPOPT_LIBRARY_PATH` ?

Comment (blegat) [reply deps/build.jl]: Thanks, fixed

**Pull request merged**